### PR TITLE
8302791: Add specific ClassLoader object to Proxy IllegalArgumentException message

### PR DESCRIPTION
**Pull request by yathamravali**
Added specific class loader object to proxy IllegalArgumentException from which the class was not visible

The bug report for the same: https://bugs.openjdk.org/browse/JDK-8302791

OpenJDK PR: https://github.com/openjdk/jdk/pull/12641

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8302791](https://bugs.openjdk.org/browse/JDK-8302791): Add specific ClassLoader object to Proxy IllegalArgumentException message

### Reviewers
 * [Mandy Chung](https://openjdk.org/census#mchung) (@mlchung - **Reviewer**)
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev pull/1802/head:pull/1802` \
`$ git checkout pull/1802`

Update a local copy of the PR: \
`$ git checkout pull/1802` \
`$ git pull https://git.openjdk.org/jdk11u-dev pull/1802/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1802`

View PR using the GUI difftool: \
`$ git pr show -t 1802`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/1802.diff">https://git.openjdk.org/jdk11u-dev/pull/1802.diff</a>

</details>

### CODE REVIEW

**Comment by yathamravali** [comment]
@mlchung - Could you kindly review the back-ported PR for jdk11? 

I've imported JavaLangAccess, SharedSecrets packages and also created a reference JLA to an object of type JavaLangAccess.

**Comment by yathamravali** [comment]
@backwaterred - Could you please add `jdk11u-fix-request` to JBS issue?

**Comment by backwaterred** [comment]
Hi @yathamravali, we typically wait for the PR to be reviewed or marked clean before adding that label (though I'm not sure it says that in the backporting document explicitly).

I also see that testing is not enabled in your fork of jdk11u-dev. Please enable it so the tests can run.

**Review by mlchung — APPROVED** [review]

**Comment by jerboaa** [comment]
As mentioned in the 17u-dev PR, approval/disapproval is done on the bug via `jdkXXu-fix-yes` or `jdkXXu-fix-no`. In the JDK 11 case, please wait until 17 backport is in.

**Comment by jerboaa** [comment]
Also please merge master and push so as to trigger GHA testing (or trigger it manually on your fork via the actions tab on github)

**Comment by yathamravali** [comment]
@jerboaa - Thanks for your inputs. Triggered GHA testing manually and JDK17 back port is pushed.

**Review by phohensee — APPROVED** [review]
Clean backport from 17u, except added missing definition of JLA in Proxy.java.

Lgtm.

**Comment by phohensee** [comment]
Tagged the JBS issue

**Comment by yathamravali** [comment]
Thanks @phohensee for reviewing. Issued `/integrate` , now is ready to be sponsored

**Comment by phohensee** [comment]
I tagged the issue with an approval request (added jdk11u-fix-request label), have to wait for a maintainer to approve before integrating. In the future, please wait until maintainer approval before adding "/integrate".

**Comment by RealCLanger** [comment]
Approved it.

/sponsor

**Pull request closed without merging**
